classes/simplemap.php

```diff
@@ -231,29 +231,29 @@ function location_search_form( $atts ) {
 
 			// Normal Field inputs.
 			$ffi['street']   = array(
-				'label' => apply_filters( 'sm-search-label-street', __( 'Street: ', 'simplemap' ), $post ),
+				'label' => apply_filters( 'sm-search-label-street', __( 'Street: ', 'SimpleMap' ), $post ),
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->